R/plotNorm.R

```diff
@@ -28,13 +29,16 @@
 #'
 #' @export
 plotNorm <- function(DGEdata,
-                     plotType = "box",
+                     plotType = "canvasXpress",
+                     plotCategory = "box",
                      normalize = "tmm") {
 
     assertthat::assert_that(any(c("matrix", "DGEobj") %in% class(DGEdata)),
                             msg = "DGEdata must be of either class 'matrix' or 'DGEobj'.")
-    assertthat::assert_that(tolower(plotType) %in% c("box", "density"),
-                            msg = "plotType must be one of 'box' or 'density'.")
+    assertthat::assert_that(plotType %in% c("ggplot", "canvasXpress"),
+                            msg = "plotType must be either ggplot or canvasXpress.")
```

Comment:
Keep canvasXpress first in message and vector both

Reply:
done